provider/gradle-mvn-push.gradle

```diff
@@ -84,6 +84,10 @@ afterEvaluate { project ->
                         }
                     }
                 }
+
+                afterEvaluate {
+                    from components.release
+                }
```

Comment:
As explained in https://developer.android.com/build/publish-library/upload-library#create-pub